src/mahoji/lib/bingo.ts

```diff
@@ -40,256 +53,231 @@ type BingoTile = (
 	},
 	{
 		id: 6,
-		name: 'Receive any Sigil from Corp',
-		oneOf: resolveItems(['Arcane sigil', 'Elysian sigil', 'Spectral sigil'])
+		name: 'Create any spirit shield from scratch',
+		oneOf: [
+			['Holy elixir', 'Spirit shield', 'Spectral sigil', 'Spectral spirit shield'],
+			['Holy elixir', 'Spirit shield', 'Arcane sigil', 'Arcane spirit shield'],
+			['Holy elixir', 'Spirit shield', 'Elysian sigil', 'Elysian spirit shield'],
+			['Holy elixir', 'Spirit shield', 'Divine sigil', 'Divine spirit shield']
+		]
 	},
 	// Row 2
 	{
 		id: 7,
-		name: 'Receive all 3 Wilderness Rings',
-		allOf: resolveItems(['Tyrannical ring', 'Treasonous ring', 'Ring of the gods'])
+		name: 'Receive any weapon from revs',
+		oneOf: resolveItems([
+			'Amulet of avarice',
+			"Craw's bow (u)",
+			"Thammaron's sceptre (u)",
+			"Viggora's chainmace (u)"
+		])
 	},
 	{
 		id: 8,
-		name: 'Receive a Tanzanite fang or Magic fang from Zulrah',
-		oneOf: resolveItems(['Tanzanite fang', 'Magic fang'])
+		name: 'Receive any TOB unique',
+		oneOf: [
+			'Scythe of vitur (uncharged)',
+			'Ghrazi rapier',
+			'Sanguinesti staff (uncharged)',
+			'Justiciar faceguard',
+			'Justiciar chestguard',
+			'Justiciar legguards',
+			'Avernic defender hilt'
+		]
 	},
 	{
 		id: 9,
-		name: 'Receive any unique seed from Gauntlet',
-		oneOf: resolveItems(['Crystal weapon seed', 'Crystal armour seed', 'Enhanced crystal weapon seed'])
+		name: 'Receive any COX unique (excluding scrolls)',
+		oneOf: resolveItems([
+			'Takon',
+			'Steve',
+			'Olmlet',
+			'Twisted bow',
+			'Elder maul',
+			'Kodai insignia',
+			'Dragon claws',
+			'Ancestral hat',
+			'Ancestral robe top',
+			'Ancestral robe bottom',
+			"Dinh's bulwark",
+			'Arcane prayer scroll',
+			'Dragon hunter crossbow',
+			'Twisted buckler'
+		])
 	},
 	{
 		id: 10,
-		name: 'Create an Odium ward or Malediction ward from scratch',
-		customReq: cl => {
-			return (
-				['Odium shard 1', 'Odium shard 2', 'Odium shard 3', 'Odium ward'].every(item => cl.has(item)) ||
-				['Malediction shard 1', 'Malediction shard 2', 'Malediction shard 3', 'Malediction ward'].every(item =>
-					cl.has(item)
-				)
-			);
-		}
+		name: 'Receive any Ignecarus unique',
+		oneOf: ['Ignis ring', 'Ignecarus dragonclaw', 'Dragon egg']
 	},
 	{
 		id: 11,
-		name: 'Receive all 3 Cerberus crystals',
-		allOf: resolveItems(['Primordial crystal', 'Pegasian crystal', 'Eternal crystal'])
+		name: 'Receive any Malygos unique',
+		allOf: ['Abyssal cape', 'Abyssal thread', 'Ori']
 	},
 	{
 		id: 12,
-		name: 'Receive/mine 12,000 Silver ore',
-		customReq(cl) {
-			return cl.amount('Silver ore') >= 12_000;
-		}
+		name: 'Receive 75 Athelas seeds',
+		receiveBank: new Bank().add('Athelas seed', 75)
 	},
 	// Row 3
 	{
 		id: 13,
-		name: 'Receive/hunt 5000 Red chinchompas',
-		customReq(cl) {
-			return cl.amount('Red chinchompa') >= 5000;
-		}
+		name: 'Receive all Polypore dungeon uniques',
+		allOf: [
```

Comment:
+ Mycelium poncho web
+ Grifolic gloves
+ Grifolic orb
Polypore stick is not on the cl